src/Console/ConsoleOutput.php

```diff
@@ -225,9 +225,12 @@ public function styleText(string $text): string
             return (string)preg_replace('#</?(?:' . $tags . ')>#', '', $text);
         }
 
+        /** @var callable $callback */
+        $callback = $this->_replaceTags(...);
```

Comment:
phpstan doesn't understand the first-class callable syntax which generates a closure 😒